apps/common/models.py

```diff
@@ -63,3 +63,39 @@ class ArchivableResource(Model):
     class Meta(TypedModelMeta):  # type: ignore[reportIncompatibleVariableOverride]
         abstract = True
         ordering = ["-id"]
+
+
+class IconEnum(models.IntegerChoices):
```

Comment:
Let's add a note that the labels should not be edited. This requires migration on the firebase as well.